igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/ScmInfoController.groovy

```diff
@@ -37,6 +38,9 @@ class ScmInfoController {
     @Autowired(required = false)
     GitHubMaster gitHubMaster
 
+    @Autowired(required = false)
+    BitBucketMaster bitBucketMaster
```

Comment:
This should be consistent with the stash support and have this under /bitbucket/ commit controller instead of joining github and bitbucket